packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -102,7 +102,7 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
 
   const onTexFieldKeyDown = useCallback(
     (ev: React.KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-      if (ev.nativeEvent.isComposing) {
+      if (ev.nativeEvent.isComposing || ev.nativeEvent.keyCode === 229 || ev.nativeEvent.which === 229) {
```

Comment:
Add a comment for why these checks are here (both the `isComposing` check and for why the extra `229` check is needed

Reply:
will do!